Assets/MixedRealityToolkit-SDK/Features/Diagnostics/DiagnosticBehavor.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Assets.MixedRealityToolkit_SDK.Features.Diagnostics
+{
+    public class DiagnosticBehavior : MonoBehaviour
+    {
+        // Use this for initialization
+        void Start()
+        {
+
```

Comment:
Let's remove empty methods

Reply:
this is a scaffolding PR, I am ok with leaving them until we know if they are needed.

Reply:
They're just the default Unity ones. Let's take'em out.